backend/src/main/java/com/eyetrade/backend/service/ArticleService.java

```diff
@@ -31,6 +31,16 @@ public ArticlesResource getArticles(){
         return  new ArticlesResource(resources);
     }
 
+    public ArticlesResource getTopArticles(){
+        List<ArticleResource> resources = new ArrayList<ArticleResource>();
+        List<Article> articles = articleRepository.findAllByOrderByScoreDesc();
+
+        for(Article article: articles){
```

Comment:
We can add null check such as collectionutils.emptyifnull(<list>)

Reply:
In this case, it returns blank list, not null. Do we still need null check? Blank list would imply no articles.

Reply:
If that way, it's okay